src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue

```diff
@@ -48,10 +65,38 @@ export default class ConsentContinuationOut extends Vue {
     return Boolean(this.filing.data?.order?.effectOfOrder)
   }
 
+  get foreignJurisdiction (): string {
+    const foreignJusrisdictionCountry = this.filing.data?.consentContinuationOut?.country
+    const country = this.getCountryName(this.filing.data?.consentContinuationOut?.country)
+    const region = this.getRegionNameFromCode(this.filing.data?.consentContinuationOut?.region)
+
+    if (foreignJusrisdictionCountry === 'CA' || foreignJusrisdictionCountry === 'US') {
+      return region + ', ' + country
+    } else {
+      return country
+    }
+  }
+
   /** Whether the filing is complete. */
   get isFilingComplete (): boolean {
     return EnumUtilities.isStatusCompleted(this.filing)
   }
+
+  private getRegionNameFromCode (short: string): string {
+    const region = this.canadaUsaRegions?.find(region => region.short === short)
+    return region?.name
+  }
```

Comment:
This is a nice, general (re-usable) function. How about moving it to the mixin? You could name it similarly to "getCountryName".

Reply:
I looked at this again and I think this one is specifically for Canada-USA regions. This getter calls the getter below canadaUsaRegions to look for region by code. On second thought, is it okay if this stays here? There is another function to get Regions by Contiry in the mixin getCountryRegions

Reply:
Oh, right. Never mind then :)